darkdetect/_mac_detect.py

```diff
@@ -6,6 +6,19 @@
 
 import ctypes
 import ctypes.util
+import subprocess
+import sys
+import os
```

Comment:
This import is no longer needed.

Reply:
I'll remove the `os` one; the other two are for the main listener function.